src/js/actions/libraries.js

```diff
@@ -64,6 +64,43 @@ define(function (require, exports) {
     };
 
     /**
+     * List of acceptable image representations that PS can place as
+     *
+     * @type {Array}
+     */
+    var _EDITABLE_IMAGE_REPRESENTATIONS = [
+        "image/png",
+        "image/jpeg",
+        "image/jpg",
+        "image/gif",
+        "image/bmp",
+        "application/photoshop",
+        "image/vnd.adobe.photoshop",
+        "application/photoshop.large",
+        "application/illustrator",
+        "application/pdf"
+    ];
+
+    /**
+     * Finds a usable representation for the image element that PS will accept
+     *
+     * @private
+     * @param {AdobeLibraryElement} element
+     * @return {AdobeLibraryRepresentation}
+     */
+    var _findPlacableImageRepresentation = function (element) {
+        var representations = element.representations;
+
+        for (var i = 0; i < representations.length; i++) {
+            if (_EDITABLE_IMAGE_REPRESENTATIONS.indexOf(representations[i].type) !== -1) {
+                return representations[i];
+            }
+        }
+            
+        throw new Error("Can't find a usable representation for image element", element.name);
```

Comment:
you might want to concatenate the strings because the second param of Error is for `filename`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error